go-controller/pkg/clustermanager/clustermanager_test.go

```diff
@@ -1173,6 +1173,7 @@ var _ = ginkgo.Describe("Cluster Manager", func() {
 
 	ginkgo.Context("Transit switch port IP allocations", func() {
 		ginkgo.It("Interconnect enabled", func() {
+			config.ClusterManager.V4TransitSwitchSubnet = "100.89.0.0/16"
```

Comment:
nit: add for v6 as well for robustness; almost always all tests cover v4 ..

Reply:
Ack